gridstatus/eia.py

```diff
@@ -658,8 +658,10 @@ def _handle_region_data(df):
 
     df["MW"] = df["MW"].astype(float)
 
-    # pivot on type
-    df = df.pivot_table(
+    df = df[df["Type"].notna()]
+
+    # pivot on Type
+    df = df.pivot(
```

Comment:
Pivot is the right choice here because we do not expect duplicates on the columns specified by `index`. `pivot` also does not drop nulls. `pivot_table` is used for aggregations of duplicates (like we do [here](https://github.com/gridstatus/gridstatus/blob/2407391dc446b8c28a327730c035266452560bd6/gridstatus/eia.py#L757-L778)).

https://pandas.pydata.org/docs/user_guide/reshaping.html#reshaping